lib/request.js

```diff
@@ -59,6 +66,34 @@ function Request(options, callback) {
   }
 }
 
+function sslCertFile() {
+  return process.env.SSL_CERT_FILE ? [process.env.SSL_CERT_FILE] : [];
+}
+
+function sslCertDir() {
+  var cert_dir = process.env.SSL_CERT_DIR;
+  if (cert_dir) {
+    return fs.readdirSync(cert_dir).map(function(f) { return path.join(cert_dir, f); });
+  } else {
+    return [];
+  }
+}
+
+function getCerts(debug) {
+  var filenames = sslCertFile().concat(sslCertDir());
+
+  if (filenames.length > 0 && debug) {
+    console.error('Adding the following trusted certificate authorities');
+  }
+
+  return filenames.map(function(filename) {
+    if (debug) {
+      console.error('  ' + filename);
```

Comment:
@dickeyxxx because hush is part of heroku-cli-util and did not want to introduce a dependency